protocol/requests/edit_message.go

```diff
@@ -2,16 +2,17 @@ package requests
 
 import (
 	"errors"
-
```

Comment:
This will give you a linting error with our CI. Putting the line back will appease the great delinter.